crewai_tools/adapters/enterprise_adapter.py

```
@@ -146,8 +147,18 @@ def tools(self) -> List[BaseTool]:
     def _fetch_actions(self):
         """Fetch available actions from the API."""
         try:
+            if (
```

should we raise an error to ensure the token is present?